packages/astro/test/fixtures/astro-response/src/components/not-found.astro

```diff
@@ -1,6 +0,0 @@
----
-return new Response(null, {
```

Comment:
That makes a lot of sense and seems unavoidable!

Reply:
I'm +1 as well, even outside of this requirement I think it makes a lot of sense on its own.

Can we make sure that there's a decent error if you try this? That should cover our migration story as well in case anyone already has.

Reply:
Added a warning for this use-case.